neptune/new/run.py

```diff
@@ -111,6 +123,7 @@ def __init__(
         self._structure: RunStructure[Attribute, NamespaceAttr] = RunStructure(NamespaceBuilder(self))
         self._lock = threading.RLock()
         self._started = False
+        self._stopped = False
```

Comment:
I would very much prefer not to have separate `_started` and `_stopped`.

Maybe `self._state` with enum value (`CREATED`, `STARTED`, `STOPPING`, `STOPPED`)?

Reply:
Can do